root/static/scripts/common/components/Autocomplete2.js

```diff
@@ -822,6 +820,7 @@ const Autocomplete2 = (React.memo(<+T: EntityItemT>(
             ? 'visible'
             : 'hidden',
         }}
+        tabIndex="-1"
```

Comment:
For some reason the other changes made it so that the `<ul>` itself was tabbable, but only for instruments and vocals (not for the relationship type or target entity). Not sure why, but it's easy enough to remove it from the tab index so that it never happens in either case.